db/migrate/20200525171520_add_time_fields_to_tests.rb

```diff
@@ -0,0 +1,6 @@
+class AddTimeFieldsToTests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :tests, :timer, :boolean, default: false
+    add_column :tests, :duration, :integer, default: nil
```

Comment:
```suggestion
    add_column :tests, :duration, :integer, default: 0
```
1. обычно дефолты стараются ставить того же типа, что и тип поля, если у тебя "интеджер", логично дефолтом ставить число
2. ставить дефолтом "нил" нет смысла в любом случае, потому что это "дефолт по дефолту" для любого поля (если дефолт не обозначен и значения в поле не записано, то там "нил" )